style.css

```diff
@@ -130,3 +130,416 @@ p {
 .menu-bar {
   display: none;
 }
+
+/* work section */
+.work {
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: 100%;
+}
+
+.work-1 {
+  display: flex;
+  flex-direction: column;
+}
+
+.work1-visual {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.work1-visual > h1 {
+  font-family: "Crete Round", serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 52px;
+  letter-spacing: 0;
+  align-items: center;
+  color: #1c1a19;
+  margin-left: 10px;
+  margin-top: 80px;
+}
+
+.line {
+  border: 1px solid black;
+  margin: 0 34px 56px 34px;
```

Comment:
Please make an effort to implement the requested changes provided by the previous reviewer. Specifically, **Please use border: thin solid black; instead of 1px solid back border.**